ldk/javascript/examples/javascript-loop/index.js

```diff
@@ -1,4 +1,5 @@
 import { clipboard, whisper } from '@oliveai/ldk';
+import { WhisperComponentType } from '@oliveai/ldk/dist/whisper/types';
```

Comment:
Is it gonna be better if `import { Markdown, Button, ... } from '@oliveai/ldk/dist/whisper/types/WhisperComponentType';`?  Then we can directly use `type: Markdown` inside our component